projects/effortless.md

```diff
@@ -0,0 +1,47 @@
+# Effortless
+
+The Effortless project includes the Habitat Scaffolding for Chef Infra and Chef InSpec and the things to make using them easy.
+
+**Slack Channel** #effortless
+
+**Triage Schedule**: TBD
+
+### Project Members
+
+#### Project Owners
+
+John Snow
+  - Github: [thelunaticscripter](https://github.com/thelunaticscripter)
+  - Slack: @thelunaticscripter
+
+Josh O'Brien
+  - Github: [OBrienCommaJosh](https://github.com/OBrienCommaJosh)
+  - Slack: @job
+
+Mark Dennard
+  - Github: [mdennard](https://github.com/mdennard)
+
+#### Project Approvers
+
+David Echols
+  - Github: [echohack](https://github.com/echohack)
+  - Slack: @echohack
+
+Jon Cowie
+  - Github: [jonlives](https://github.com/jonlives)
+
+Scott Ford
+  - Github: [scottford-io](https://github.com/scottford-io)
+
+Susan Evans
+  - Github: [susanev](https://github.com/susanev)
+
+#### Project Reviewers
+
+Robb Kidd
```

Comment:
Robb you are currently assigned an an approver, and I have not received a request to make you a reviewer. Did you want to increase your responsibility on effortless? Or is this a misnomer?

Reply:
Reviewers: have opinions
Approvers: can approve opinions

I do not believe my responsibilities in the project are to approve opinions. My contributions are in the flavor of having opinions for Approvers and Owners to consider.

Reply:
roger, I'll move you to reviewers